validate/add-a-validator.md

```diff
@@ -14,6 +14,9 @@ After successfully bootstrapping your node, you may add it as a validator on the
 * The minimum staking period is 14 days on Mainnet and on Socotra Testnet. The maximum staking period for both networks is 365 days.
 * Any delegator staking their funds on your node will pay you the delegation fee, which is 12% of their reward. Delegators can stake no more than x4 of your own stake.
 
+> [!WARNING]  
+> After Creating a Validator Delegation, you can NOT do a validate tx to the same node again, you are also unable to increase the time or the stake until the end of your validation period.
```

Comment:
Hello,

This warning is interesting and useful.
Could you please reformulate the sentence to make it clearer?

Thanks

Reply:
reworded it.